cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/queue/JobQueueDataset.java

```diff
@@ -179,16 +180,7 @@ private boolean isTriggerSatisfied(Trigger trigger, List<Notification> notificat
       // is initially created
```

Comment:
Shouldn't this code be removed now? TimeTrigger implements `SatisfiableTrigger` right?